lib/src/frame.dart

```diff
@@ -17,12 +17,36 @@ final _vmFrame = RegExp(r'^#\d+\s+(\S.*) \((.+?)((?::\d+){0,2})\)$');
 //     at VW.call$0 (eval as fn
 //         (https://example.com/stuff.dart.js:560:28), efn:3:28)
 //     at https://example.com/stuff.dart.js:560:28
-final _v8Frame =
+final _v8JsFrame =
     RegExp(r'^\s*at (?:(\S.*?)(?: \[as [^\]]+\])? \((.*)\)|(.*))$');
 
 // https://example.com/stuff.dart.js:560:28
 // https://example.com/stuff.dart.js:560
-final _v8UrlLocation = RegExp(r'^(.*?):(\d+)(?::(\d+))?$|native$');
+//
+// Group 1: URI, required
+// Group 2: line number, required
+// Group 3: column number, optional
+final _v8JsUrlLocation = RegExp(r'^(.*?):(\d+)(?::(\d+))?$|native$');
+
+// With names:
+//
+//     at Error.f (wasm://wasm/0006d966:wasm-function[119]:0xbb13)
+//     at g (wasm://wasm/0006d966:wasm-function[796]:0x143b4)
+//
+// Without names:
+//
+//     at wasm://wasm/0005168a:wasm-function[119]:0xbb13
+//     at wasm://wasm/0005168a:wasm-function[796]:0x143b4
+//
+// Matches named groups:
+//
+// - "member": optional, `Error.f` in the first example, NA in the second.
+// - "uri":  `wasm://wasm/0006d966`.
+// - "index": `119`.
+// - "offset": (hex number) `bb13`.
+final _v8WasmFrame = RegExp(r'^\s*at (?:(?<member>\S+) )?'
+    r'(?:\(?(?:(?<uri>wasm:\S+):wasm-function\[(?<index>\d+)\]'
+    r'\:0x(?<offset>[0-9 a-f A-F]+))\)?)$');
```

Comment:
Maybe say that this optimistically assumes that there won't be an unmatched start/end parentheses, or that only the two formats above will actually occur. That is, this RegExp also matches
```
    at Something.other wasm://wasm/0006d966:wasm-function[119]:0xbb13
```
which is neither of the format shown above.

It's probably safe because no other stack trace will contain both `wasm://wasm` and `:wasm-function[\d+]`, so only Wasm stack traces will be recognized, and they won't have weird formats. Just be explicit about any shortcuts taken, so a reader won't get confused and think that you *want* to match those other formats. It's just that you don't care to avoid it, since they won't happen.

That might be important for someone maintaining this in the future.
(A lot of words to say that RegExps are not self-documenting! I prefer to always have a prose description of what it intends to match. Too often I have tried to change a RegExp without knowing what it *intended*, and it was not always what it was actually doing.)